src/backend/functions/change-requests-review.ts

```diff
@@ -23,28 +27,20 @@ export const reviewChangeRequest: Handler = async ({ body }, _context) => {
   if (!foundCR) return buildNotFoundResponse('change request', `#${crId}`);
 
   // update change request
-  const update = prisma.change_Request.update({
+  const update = await prisma.change_Request.update({
     where: { crId },
     data: { reviewNotes, accepted, dateReviewed: new Date() }
   });
 
   // TODO: consider transformer or other possible return body
-  return buildSuccessResponse(update);
+  return buildSuccessResponse({ message: `Change request #${update.crId} successfully reviewed.` });
```

Comment:
Good enough for this MVP, would like to have a place to store messages so we do not have to hardcode it in the file.